editoast/src/models/prelude/mod.rs

```diff
@@ -63,9 +64,13 @@ pub type Row<M> = <M as Model>::Row;
 /// A type alias for the [Model::Changeset] associated type
 ///
 /// Helps silent compiler errors about type amibiguity.
-#[allow(unused)]
 pub type Changeset<M> = <M as Model>::Changeset;
 
+/// A type alias for the [Model::Error] associated type
+///
+/// Helps silent compiler errors about type amibiguity.
```

Comment:
```suggestion
/// Helps silent compiler errors about type ambiguity.
```

Hmm, and by the way, I believe this is copy/paste from above so it might need to be corrected above too.

Reply:
600bd2ca8